pom.xml

```diff
@@ -339,6 +352,9 @@
                         <exclude>e6e4ebea-da12-4bde-8f24-6272925ad093</exclude>
                         <exclude>4a7955ac-037c-42bc-a83e-b51efe2490af</exclude>
                         <exclude>d3477f9c-032a-44a7-a5e1-02ae35e4737c</exclude>
+                        <exclude>c998ada9-b8ca-4da9-ab6c-e863d6ef80e5</exclude>
+                        <exclude>fb558ee6-6701-41b7-bc85-b298e2e2c0e0</exclude>
+                        <exclude>8923f062-a77b-48ca-80a2-d3af45919229</exclude>
```

Comment:
🙄

Reply:
Yeah, unfortunately needed for log4j in tests